inventory-sample.yml

```
@@ -14,7 +14,12 @@ k3s_cluster:
     ansible_port: 22
     ansible_user: debian
     k3s_version: v1.26.9+k3s1
-    token: "mytoken"  # Use ansible vault if you want to keep it secret
+    # The token must be a random string of reasonable randomness. You can generate
```

```suggestion
    # The token should be a random string of reasonable randomness. You can generate
```
If people want bad security, they can have it.

---

nitpicking a bit imho, but okay